### PR TITLE
Update of rtl_433_ESP to version 0.1.3 - Support for SX127X transceiver module

### PR DESCRIPTION
**Pull request by NorthernMan54**
## Description:

The main features of the upgrade to 0.1.3 of rtl_433_ESP are

1 - Change Transceiver Library to [RadioLib](https://github.com/jgromes/RadioLib)

2 - Add support for SX127X transceiver module ( I used the 433 Mhz version of this [Heltec module](https://heltec.org/project/wifi-lora-32/) for development ).

3 - Added automatic calibration of the RSSI Signal detection Threshold

4 - Removed dependency on loop for start and end of signal detection, and moved to a ESP32 Task

5 - Moved received signal decoding to a ESP32 Task

6 - Identified Blueline PowerCost Monitor decoder as source of significant memory overhead and disabled.

7 - Multi Receiver support has been validated with RF, RF2 and PiLight

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] I accept the [DCO](https://github.com/1technophile/OpenMQTTGateway/blob/development/docs/participate/development.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by DigiH** [comment]
Great news @NorthernMan54! 

Hope it's ok to turn this PR into a draft for the time being, so no accidental merge is guaranteed and you can continue the work on it.

**Comment by 1technophile** [comment]
Ordering a heltec Lora 433mhz now !

**Comment by 1technophile** [comment]
Received the Heltec and started testing, here is my feedback, it worked very quickly after uploading your proposed configuration `[env:heltec-rtl_433]
`
Here are the devices that I have working:
SENSORS:
* Digoo DG-R8S
```
{"model":"Prologue-TH","subtype":9,"id":22,"channel":1,"battery_ok":1,"temperature_C":26.2,"humidity":52,"button":0,"protocol":"Prologue, FreeTec NC-7104, NC-7159-675 temperature sensor","rssi":-65,"duration":933996}
```
* YC-2000B plug remote control reception
```
{"model":"Smoke-GS558","id":21584,"unit":10,"learn":0,"code":"2a8a0a","protocol":"Wireless Smoke and Heat Detector GS 558","rssi":-41,"duration":316996}
```
* ANKUOO remote control
```
{"model":"Generic-Remote","id":17733,"cmd":84,"tristate":"Z0ZZZ0ZZZZZ0","protocol":"Generic Remote SC226x EV1527","rssi":-48,"duration":298996}
```
* WS2032 weather station
```
{"model":"WS2032","id":37525,"battery_ok":1,"temperature_C":26,"humidity":52,"wind_dir_deg":90,"wind_avg_km_h":1.548,"wind_max_km_h":13.932,"rain":256,"flags":0,"mic":"CRC","protocol":"WS2032 weather station","rssi":-49,"duration":407996}
```

* Digoo R8H
* Digoo compact PIR

Very good @NorthernMan54 !

File: main/ZgatewayRTL_433.ino [hunk 1]
```diff
@@ -49,8 +44,9 @@ void rtl_433_Callback(char* message) {
     return;
   }
 
+  unsigned long MQTTvalue = (int)RFrtl_433_ESPdata["id"] + round(RFrtl_433_ESPdata["temperature_C"]);
   String topic = String(subjectRTL_433toMQTT);
-#  if valueAsATopic
+#  ifdef valueAsATopic
```
Comment (DigiH) [inline main/ZgatewayRTL_433.ino]: Hi @NorthernMan54,

we were trying to get away from the sometimes confusing 'just defined' to go for a correct true/false with https://github.com/1technophile/OpenMQTTGateway/pull/1260, so here it should remain **if**.

Thanks

**Comment by NorthernMan54** [comment]
@DigiH @1technophile - This should be ready for merging, only outstanding item is updating the openMQTTGateway documentation

**Pull request closed without merging**